lib/pbxProject.js

```diff
@@ -338,32 +338,43 @@ pbxProject.prototype.addFramework = function(fpath, opt) {
     file.fileRef = this.generateUuid();
     file.target = opt ? opt.target : undefined;
 
-    if (this.hasFile(file.path)) return false;
-
-    this.addToPbxBuildFileSection(file);        // PBXBuildFile
-    this.addToPbxFileReferenceSection(file);    // PBXFileReference
-    this.addToFrameworksPbxGroup(file);         // PBXGroup
+    var fileReference = this.hasFile(file.path);
+    if (fileReference) {
+        var key = this.getFileKey(file.path);
+        file.fileRef = key;
+    } else {
+        this.addToPbxFileReferenceSection(file);    // PBXFileReference
+        this.addToFrameworksPbxGroup(file);         // PBXGroup
```

Comment:
Maybe we need this as well
```
this.addToPbxBuildFileSection(file);
```

Reply:
I don't think we need to do that if we won't add it to the FrameworksBuildPhase.